src/shared/featureFlags/featureFlag.ts

```diff
@@ -36,12 +36,12 @@ export function useFlags(fallback) {
         'Warning! Self hosted build is missing a default feature flag value.'
       )
     }
-    return fallback
+    return fallback ?? {}
```

Comment:
nit: could set this to the initial param as well since fallback is optional to begin with